apps/coordinator/src/reducers/transactionReducer.js

```diff
@@ -280,16 +284,18 @@ function finalizeOutputs(state, action) {
   // First try to build the transaction via PSBT, if that fails (e.g. an input doesn't know about its braid),
   // then try to build it using the old TransactionBuilder plumbing.
   try {
-    const unsignedTransactionPSBT = unsignedMultisigPSBT(
-      state.network,
-      state.inputs,
-      state.outputs,
-    );
-    unsignedTransaction = unsignedTransactionObjectFromPSBT(
-      unsignedTransactionPSBT,
+    const args = {
+      network: state.network,
+      inputs: state.inputs.map(convertLegacyInput),
+      outputs: state.outputs.map(convertLegacyOutput),
+    };
+    const psbt = getUnsignedMultisigPsbtV0(args);
+    unsignedTransaction = Transaction.fromHex(
```

Comment:
Another example of not needing a specialized function. If we have a psbt, it has the unsignedTx on its globalMap (if v0) so just use that.